src/shared_gui_components/OSGridView.cpp

```diff
@@ -78,6 +78,8 @@
 #  define HEIGHT 60
 #endif
 
+const int NUM_ROWS_PER_GRIDLAYOUT = 5;
```

Comment:
This is set low right now for testing, I will probably bump it to 51 for real use.

Reply:
```suggestion
constexpr int NUM_ROWS_PER_GRIDLAYOUT = 5;
```

Reply:
if I recall correctly, the original design was to have 50 rows per grid layout. Did you change that when you refactored?

Anyhoo, if we're displaying X by X, is there any way we can do that multithreaded?